shared/src/test/java/io/pravega/shared/segment/SegmentToContainerMapperTests.java

```diff
@@ -42,13 +42,13 @@ public void testConstructor() {
      */
     @Test
     public void testUniformMapping() {
-        testUniformMapping(20, 10000, 0.10, (num) -> getSegmentName(num));
-        testUniformMapping(20, 10000, 0.10, (num) -> Integer.toString(num));
-        testUniformMapping(100, 100000, 0.10, (num) -> getSegmentName(num));
-        testUniformMapping(100, 100000, 0.10, (num) -> Integer.toString(num));
-        testUniformMapping(100, 100000, 0.10, (num) -> Integer.toBinaryString(num));
-        testUniformMapping(100, 100000, 0.10, (num) -> Integer.toOctalString(num));
-        testUniformMapping(100, 100000, 0.10, (num) -> Integer.toHexString(num));
+        testUniformMapping(20, 10000, 0.10, num -> getSegmentName(num));
```

Comment:
This change does not seem to be related to the upgrades proposed here, if that's the case and for the future, let's try to avoid unrelated formatting changes.

Reply:
It is related, as are the similar changes above. The new version of checkstyle considers it an error to surround  a single parameter in a lambda with parenthesis. (I don't know why, but it does.) So I'm removing the parenthesis